hazelcast/src/main/resources/hazelcast-default.yaml

```diff
@@ -324,3 +324,13 @@ hazelcast:
 #    locks:
 #      default:
 #        lock-acquire-limit: 0
+
+  metrics:
+    enabled: true
+    management-center:
+      enabled: true
+      retention-seconds: 5
+    jmx:
+      enabled: true
+    collection-frequency-seconds: 5
+    data-structure-metrics-enabled: false
```

Comment:
Do we have `data-structure-metrics-enabled` property? I thought it's only possible to override the default (which is `true`) with the sys prop now.

Reply:
Thanks, it's a leftover that was ignored by the config processor. Removed.